server/src/main/java/jake2/server/SV_MAIN.java

```diff
@@ -513,16 +511,17 @@ void SV_SendClientMessages() {
                 continue;
             // if the reliable message overflowed,
             // drop the client
-            if (c.netchan.message.overflowed) {
-                c.netchan.message.clear();
-                c.datagram.clear();
+            int pendingReliableSize = c.netchan.reliable.stream().mapToInt(NetworkMessage::getSize).sum();
```

Comment:
yes, every time we need to know the size of accumulated messages, need to iterate over the whole collection.
most of the time it's not a big deal however